packages/backend-core/src/events/publishers/backup.ts

```diff
@@ -10,3 +16,16 @@ export async function appBackupRestored(backup: AppBackup) {
 
   await publishEvent(Event.APP_BACKUP_RESTORED, properties)
 }
+
+export async function appBackupTriggered(
+  appId: string,
+  trigger: AppBackupTrigger,
+  name?: string
+) {
+  const properties: AppBackupTriggeredEvent = {
+    appId: appId,
+    backupName: name,
```

Comment:
Should we add the `AppBackupType` as well? So we can distinguish real backups from restores